_dashboards/dev-tools/run-queries.md

```diff
@@ -9,44 +9,36 @@ redirect_from:
 
-Write your queries in the editor pane on the left side of the console:
+An example of the query and response panes is shown in the following image.
 
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-request.png" alt="Request pane">{: .img-fluid }
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/query-request-ui.png" alt="Console UI with query and request">
 
-Collapse or expand your query by selecting the triangle next to the line numbers.
-{: .tip}
+### Query options 
 
-To learn more about writing queries in OpenSearch domain-specific language (DSL), see [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl).
+When writing queries using the console, there are common features that can help you write queries more efficiently and accurately. The following table describes these features and how you can use them.
 
-### Comments
-
-Use `#` at the beginning of a line to write single-line comments.
-
-### Autocomplete
-
-OpenSearch provides autocomplete suggestions for fields, indexes and their aliases, and templates. To configure autocomplete preferences, update them in [Console Settings](#updating-the-console-settings).
-
-## Sending the request 
-
-To send a query to OpenSearch, select the query by placing the cursor anywhere in the query text. Then choose the play icon ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/play-icon.png" class="inline-icon" alt="play icon"/>{:/}) on the upper right of the request or press `Ctrl/Cmd+Enter`:
-
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-send.png" alt="Send request">
-
-OpenSearch displays the response in the response pane on the right side of the console:
-
-<img src="{{site.url}}{{site.baseurl}}/images/dev-tools/dev-tools-response.png" alt="Response pane">{: .img-fluid }
+Feature | Description | How to use |
+--------|-------------|------------|
+**Collapsing or expanding a query** | Helpful to use when you have a long query or only want to focus on a specific part of the query. | To hide or show details of your query, select the expander arrow ({::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/arrow-down-icon.png" class="inline-icon" alt="arrow down icon"/>{:/}) next to the line number. |
+**OpenSearch search language** | Helpful to use when you need flexibility in writing complex queries. | To learn how to use the OpenSearch query domain-specific language (DSL), see [Query DSL]({{site.url}}{{site.baseurl}}/opensearch/query-dsl/). |
```

Comment:
This is the only outlier here because it's not an action. Maybe move it to before the table as a sentence?

Reply:
Also, it's pretty much the only option in DevTools, right? Except for sending everything as a query parameter. So "helpful to use" does not apply.

Reply:
Revised. See lines 1-2.